frontend/lib/ifixit-api/devices.ts

```diff
@@ -0,0 +1,53 @@
+import { IFIXIT_ORIGIN } from '@config/env';
+import { getDeviceHandle } from '@models/product-list';
```

Comment:
What do you think about moving the implementation of `getDeviceHandle` in this new file so we avoid a circular dependency?

Reply:
Good call, Fixed!